app/src/main/java/com/codesoom/assignment/global/utils/TimeUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.codesoom.assignment.global.utils;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+/**
+ * 시간 처리를 담당합니다.
+ */
+public class TimeUtil {
+    private TimeUtil() {
+    }
+
+    /**
+     * LocalDatetime 값을 Date로 변환해 리턴합니다.
+     *
+     * @param localDateTime 입력받은 시간
+     * @return 변경된 시간 포맷
```

Comment:
시간이 변경된 것으로 잘못 읽을 가능성이 있어 보입니다.
```suggestion
     * @return 포맷이 변경된 시간
```

Reply:
다르게 해석될 여지가 있네요 !